src/pages/staff/dashboard/LearningPage.jsx

```
@@ -558,21 +718,176 @@ export default function LearningPage() {
+      const first = items[0];
+      const last = items[items.length - 1];
+      if (e.shiftKey) {
+        if (document.activeElement === first) { e.preventDefault(); last.focus(); }
+      } else {
+        if (document.activeElement === last) { e.preventDefault(); first.focus(); }
+      }
+    };
+
+    if (!open) return null;
+    return (
+      <div className="fixed inset-0 z-50 bg-black/40" onClick={() => { setOpen(false); try { track('search_close', { page: 'learning' }); } catch { /* no-op */ } try { setTimeout(() => document.querySelector('[data-testid=\"bottomnav-search\"]')?.focus(), 0); } catch { /* no-op */ } }}>
```

ESLint

[error] Unnecessary escape character: \".

(no-useless-escape)